R/plot.R

```diff
@@ -97,26 +97,28 @@ plotSamples <- function(samples, groups=NULL, colors=NULL, gene=NULL, embedding.
     groups <- as.factor(groups)
   }
 
-  if (is.null(embedding.type)) {
-    embedding.type <- if ('seurat' %in% class(samples[[1]])) "tsne" else "tSNE"
+  if (is.null(x = embedding.type)) {
+    embedding.type <- if (inherits(x = samples[[1]], what = 'seurat')) {
+      'tsne'
+    } else if (inherits(x = samples[[1]], what = 'Seurat')) {
+      'umap'
```

Comment:
It's the last part. Please, put tsne here.

Reply:
Done